spark/notebooks/Iceberg - Getting Started.ipynb

```diff
@@ -39,7 +58,11 @@
   {
    "cell_type": "markdown",
    "id": "747bee98",
-   "metadata": {},
+   "metadata": {
```

Comment:
nit: I'm guessing that's probably coming from pycharm right? is this something we'd want to have here?

Reply:
Great point, I've removed it 👍🏻

Reply:
Was a bit too enthusiastic about removing the metadata. Also added `nbstripout` to strip all the output of the notebooks, and it will also check if the notebooks are valid.